Fix:

conftest.c:491:23: error: implicit declaration of function 'utime' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- configure.orig	2021-01-09 03:57:15.000000000 -0600
+++ configure	2021-01-20 04:36:06.000000000 -0600
@@ -49393,7 +49393,7 @@
   cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 #include <stddef.h>
-                 #include <time.h>
+#include <utime.h>
 
 int
 main (void)
